lib/signaling/v2/livetranscriptionsignaling.js

```diff
@@ -12,18 +13,25 @@ class LiveTranscriptionSignaling extends MediaSignaling {
   constructor(getReceiver, options) {
     super(getReceiver, 'extension_transcriptions', options);
 
+    this.isEnabled = false;
+
     this.on('ready', transport => {
+      this.isEnabled = true;
+
       transport.on('message', message => {
         switch (message.type) {
           case 'extension_transcriptions':
-            // TODO: Test and update the message format once this is ready on the backend
```

Comment:
@charliesantos Let me know if you were thinking of doing any transformation to the message before bubbling it up

Reply:
With the current implementation, what is the JSON output below?
```javascript
room.on('liveTranscription', console.log);
```

Reply:
Okay, on it.